### ISSUE TITLE
Some fixes to labels and comments

### ISSUE THREAD

**Issue by dominicbeesley** [raybellis/mos120#1]
Hi Ray,

I've got a patch to the combined listing - I'm not sure of the best way to get this back into the source-sources as it were. Mainly some fixes to the confusing labels used in the micronet sources in the area of SOUND/ENVELOPE processing which I've been tinkering with lately.

Thanks for doing this - it has been very useful!

[patch-20200831.zip](https://github.com/raybellis/mos120/files/5149836/patch-20200831.zip)

**Comment by raybellis** [comment]
Hmm, tricky.   I've confirmed that your resulting `mos120.s` does compile to the expected binary, but ultimately that file is supposed to be the _output_ of this repository, not the input.

Ideally I think the solution would be for someone (yourself?) to work your patches into the individual files that make up the `input` and `src` directories, so that running my `compile.sh` script eventually produces the same output as your patched `mos120.s`.    Perhaps also JGH would consider then putting those online on his archive.

**Comment by raybellis** [comment]
oh, hang on.   It looks like at some point I started treating `mos120.s` as the "definitive" source, since there are changes made there to make compilation (and modification) easier that aren't in the original source files.   This makes it even trickier :(

**Comment by dominicbeesley** [comment]
Yes, I couldn't work out what / where to change. So I sent the patch! 

It's not a big deal but some of the (micronet?) labels and comments in the sound section had me going round in circles!

**Comment by raybellis** [comment]
I've created a branch that has your patch applied - https://github.com/raybellis/mos120/tree/beesley

Daniel Jameson suggested that you mention your changes on the original thread on Stardot.

**Comment by dominicbeesley** [comment]
Sorry Ray i just spotted this.  I'll party sonething up tonight or
tomorrow. Is there any reason to keep it on a branch?

**Comment by raybellis** [comment]
I'd quite like someone else more familiar with the code than I am to review your changes before merging them into the main branch since they'll cause a significant divergence from the original assembly files.    They'll need to ignore whitespace changes, though...

**Comment by dominicbeesley** [comment]
Fair enough. I'll double check but I'm pretty sure it compiles to a byte
exact copy.

**Comment by raybellis** [comment]
oh, it does - I checked that myself.   It was more the semantics of the sound variables, and having someone else pipe up and say "how did we never spot that" :)
